Sources/MessagingInApp/Views/InlineMessageBridgeView.swift

```diff
@@ -0,0 +1,169 @@
+///
+/// Usage:
+/// ```swift
+/// let bridgeView = InlineMessageBridgeView()
+/// bridgeView.attachToParent(parent: parentView, delegate: delegateImpl)
+/// bridgeView.elementId = "elementId"
+/// ```
+public class InlineMessageBridgeView: UIView, InlineMessageViewProtocol {
+    /// The element ID used to identify which inline message to display.
+    /// Setting this property triggers view setup.
+    public var elementId: String? {
+        didSet {
+            setupInlineMessageView()
+        }
+    }
+
+    /// Delegate for handling wrapper-specific events and interactions.
+    public weak var delegate: InlineMessageBridgeViewDelegate?
+
+    /// Cached width of the last rendered message to prevent unnecessary updates.
+    private var lastRenderedWidth: CGFloat?
+
+    /// Cached height of the last rendered message to prevent unnecessary updates.
+    private var lastRenderedHeight: CGFloat?
+
+    /// Initializes a new inline message bridge view.
+    public init() {
+        super.init(frame: .zero)
+    }
+
+    /// InlineMessageBridgeView does not support Interface Builder initialization.
+    /// Use `init()` instead.
+    required init?(coder: NSCoder) {
+        assertionFailure(
+            "InlineMessageBridgeView does not support initialization from Interface Builder. Use init() instead."
+        )
+        return nil
+    }
+
+    deinit {
+        onViewDetached()
+    }
+
+    /// Attaches this view to a parent view and sets the delegate.
+    /// Called once from wrapper constructor to establish the view hierarchy.
+    /// - Parameters:
+    ///   - parent: The parent view to attach this wrapper to
+    ///   - delegate: The delegate for handling wrapper events
+    public func attachToParent(parent: UIView, delegate: InlineMessageBridgeViewDelegate) {
+        self.delegate = delegate
+        constrainToFillParent(parent)
+    }
+
+    /// Called every time the wrapper view is attached to a screen.
+    /// May be called multiple times during the wrapper's lifecycle.
+    public func onViewAttached() {
+        setupInlineMessageView()
+    }
+
+    /// Called every time the wrapper view is detached from a screen.
+    /// Cleans up resources to prevent memory leaks.
+    public func onViewDetached() {
+        inAppMessageView?.teardownView()
+    }
+
+    /// Sets up the underlying Gist inline message view if needed.
+    private func setupInlineMessageView() {
+        guard let elementId else {
+            return // No element ID set, nothing to setup
+        }
+
+        // If view already exists, just update the element ID
+        if let existingView = inAppMessageView {
+            existingView.elementId = elementId
+            return
+        }
+
+        // Create new inline message view
+        let inlineInAppMessageView = createGistMessageView(elementId: elementId)
+        inlineInAppMessageView.constrainToFillParent(self)
+        onNoMessageToDisplay()
```

Comment:
why are we sending this on init?

Reply:
I initially thought this would ensure we start with `0` height, but that's not really needed, removing it.